docs/modules/ROOT/pages/documentation/microprofile/metrics/vendor-metrics.adoc

```diff
@@ -1,7 +1,7 @@
 [[custom-vendor-metrics]]
 = Custom Vendor Metrics
 
-JMX Mbeans can be exposed as ***Custom Vendor Metrics*** by the user by supplying a custom https://github.com/payara/Payara/blob/f30de88abca76111caadf0d57249d77d53b067e7/appserver/payara-appserver-modules/microprofile/metrics/src/main/resources/metrics.xml[metrics.xml] file in the `${PAYARA_HOME}/glassfish/domains/${DOMAIN_NAME}/config/` folder with the following format:
+xref:documentation/user-guides/monitoring/mbeans.adoc[JMX Mbeans] can be exposed as ***Custom Vendor Metrics*** by the user by supplying a custom https://github.com/payara/Payara/blob/f30de88abca76111caadf0d57249d77d53b067e7/appserver/payara-appserver-modules/microprofile/metrics/src/main/resources/metrics.xml[metrics.xml] file in the `${PAYARA_HOME}/glassfish/domains/${DOMAIN_NAME}/config/` folder with the following format:
```

Comment:
Note that this metrics.xml file will result in a NPE when used as {domain}/config/metrics.xml file as it is missing the <tag> which is required since MP Metrics 2.0.

Reply:
Fixed by linking to the correct metrics.xml file